script/__snapshots__/migrate-test-e2e.js.snap

```diff
@@ -191,7 +191,9 @@ exports[`expected file changes > cspell.json 1`] = `
  		"tsup",
 -		"Unstaged",
 -		"vitest",
- 		"wontfix"
+-		"wontfix",
+-		"precommit"
++		"wontfix"
```

Comment:
[Style] Super nitty nit: could you keep these alphabetized please? They're currently alphabetized and it brings me joy to keep them that way. 😄